### DISCUSSION TITLE
Data for France

### DISCUSSION THREAD

**Discussion by Philnext** [owid/covid-19-data#1942] [Q&A] [answered]
Hello
I try to uderstand some differences in data for France.
ex : for 2021:09:06 column **people_fully_vaccinated** : 
In owid data : https://github.com/owid/covid-19-data/blob/master/public/data/vaccinations/country_data/France.csv : **41653844**
in the source : https://www.data.gouv.fr/fr/datasets/r/efe23314-67c4-45d3-89a2-3faef82fae90 : **45664530** 

The column **people_fully_vaccinated** is OK (same value : **49070298**).

**Comment by lucasrodes** [comment, marked as answer]
See https://github.com/owid/covid-19-data/issues/1791
